server/app.js

```diff
@@ -1,17 +1,13 @@
 import express from 'express';
```

Comment:
we should decide which format we'll use for imports. I'm ok with sticking with the ES5 require style as it's supported natively by NodeJS

also all the other imports are in the require format.